recipe/meta.yaml

```diff
@@ -37,7 +37,7 @@ test:
 
 about:
   home: http://www.scitools.org.uk/iris
-  license: LGPL-3.0
+  license: LGPL-3.0-only
```

Comment:
I'm not a lawyer and I don't understanding anything about licenses but shouldn't this be or-later?

https://github.com/SciTools/iris/blob/master/COPYING#L584-L587

Reply:
I really don't know, it's always confused me, but then again I am rather dense...
I'll change it to `-or-later` what the heck...

Reply:
I'm never really certain about this as well :smile:
If you want to be conservative, and leave `-only` until we figure this out, that would be fine by me.